src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -747,4 +747,10 @@ protected boolean checkScriptPresentInElement(WebElement element, String scriptT
     return false;
   }
 
+  public void clickOnArticleLink(String linkName) {
+    WebElement link = driver.findElement(
+        By.cssSelector("a[title='" + linkName + "']"));
+    link.click();
```

Comment:
We are waiting for element before click in `findElement`:
>This method is affected by the 'implicit wait' times in force at the time of execution. The findElement(..) invocation will return a matching row, or try again repeatedly until the configured timeout is reached
Source: https://selenium.googlecode.com/git/docs/api/java/org/openqa/selenium/WebDriver.html#findElement-org.openqa.selenium.By-

Maybe, you meant waiting for element to be clickable, however, I tested this solution on Chrome and FF, locally and on VMs, on Mercury and Oasis and there are no issues with just click.